### ISSUE TITLE
Radix doesn't recognice path with "s" at end

### ISSUE THREAD

**Issue by faustinoaq** [luislavena/radix#21]
Based on @msa7 message on [Gitter](https://gitter.im/sdogruyol/kemal?at=58c39f5ee961e53c7fcda709)

Radix version: 0.37
Crystal version: 21.1 Linux x86_64

```crystal
require "radix"

tree = Radix::Tree(Symbol).new
tree.add "/products", :products
tree.add "/product/new", :product_new

result = tree.find "/products"

if result.found?
  puts result.payload
else
  puts "Not found"
end

# => Not found
```

**Comment by luislavena** [comment]
Thank you @faustinoaq for the report. :heart: :heart: :heart: 

Will look into this over the weekend.

Cheers.

**Comment by luislavena** [comment]
Hello @faustinoaq, I've just merged #22 that corrects this issue.

Will wrap that and other changes and perform a new release of Radix shortly.

Thank you for your report and your help making Radix better! :heart: :heart: :heart:
